lib/ain-grpc/src/impls.rs

```diff
@@ -70,3 +70,47 @@ impl TryFrom<TransactionV2> for EthTransactionInfo {
         })
     }
 }
+
+impl TryFrom<&str> for EthPendingTransactionInfo {
+    type Error = TransactionError;
+
+    fn try_from(raw_tx: &str) -> Result<Self, Self::Error> {
+        let signed_tx: SignedTx = raw_tx.try_into()?;
+
+        let to = if let Some(signed_to) = signed_tx.to() {
+            String::from("0x") + &hex::encode(signed_to.as_fixed_bytes())
+        } else {
+            String::from("null")
+        };
+
+        let input = if !signed_tx.data().is_empty() {
+            format!("0x{}", hex::encode(signed_tx.data()))
+        } else {
+            String::from("0x0")
+        };
+
+        let pending_transaction = EthPendingTransactionInfo {
+            hash: format!(
+                "0x{}",
+                hex::encode(signed_tx.transaction.hash().as_fixed_bytes())
+            ),
```

Comment:
Wonder if we can simplify these. 

```suggestion
            hash: format_hash(signed_tx.transaction.hash()),
```